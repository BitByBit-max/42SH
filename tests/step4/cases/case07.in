#!/bin/bash

input_string="goodbye"

case $input_string in
    # match a string that starts with "hello"
    hello*)
        echo "The string starts with 'hello'"
        ;;

    # match a string that is exactly "goodbye"
    goodbye)
        echo "The string is exactly 'goodbye'"
        ;;

    # match any other string
    *)
        echo "The string does not match any of the other cases"
        ;;
esac
