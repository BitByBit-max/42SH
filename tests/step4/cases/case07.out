The string is exactly 'goodbye'
